apps/src/netsim/NetSimPacketEditor.js

```diff
@@ -117,7 +117,7 @@ var NetSimPacketEditor = module.exports = function (initialConfig) {
    * @type {Number}
    * @private
    */
-  this.maxPacketSize_ = initialConfig.maxPacketSize || Infinity;
+  this.maxPacketSize_ = initialConfig.maxPacketSize || 8192;
```

Comment:
I feel that both values are pretty conservative; we didn't start seeing problems until people were sending messages > 50k. I feel that with 8k characters, people are only going to hit the limit if they really are just copy/pasting in a bunch of characters. It's reasonable that people are going to hit 1k with a typed message, and I'd rather allow that where possible.

Anyone else have thoughts? @islemaster @sfilman @alicesteinglass